app/upgrades/v17/upgrades.go

```diff
@@ -15,14 +15,7 @@ func CreateUpgradeHandler(
 	configurator module.Configurator,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
-		logger := ctx.Logger().With("upgrade", UpgradeName)
-
-		// revenue module is deprecated
-		logger.Debug("deleting revenue module from version map...")
-		delete(vm, "revenue")
-
 		// Leave modules as-is to avoid running InitGenesis.
-		logger.Debug("running module migrations ...")
```

Comment:
Why are we removing the logic for V17?

Reply:
because that logic was not included in the v17 release.